x-pack/plugin/core/src/main/resources/new-security-index-template.json

```diff
@@ -0,0 +1,151 @@
+{
+  "index_patterns" : [ ".security-*" ],
```

Comment:
we should probably exclude the tokens index from getting this template

Reply:
We don't use these files as templates anymore, so the `index_pattern` field is only for doc purposes only. I agree it is confusing, I will change it.